src/OpenFin.FDC3.Tests/obj/Debug/OpenFin.FDC3.Tests.csproj.CoreCompileInputs.cache

```diff
@@ -0,0 +1 @@
+c0551a6e366d34b412324f2e6dc36e4ffb02c557
```

Comment:
This project appears to be in the incorrect directory, and thus is not getting the correct `.gitignore` applied to it.

Reply:
This has been resolved on another branch